sefaria/system/exceptions.py

```diff
@@ -99,3 +99,13 @@ def __init__(self, method):
         self.method = method
         self.message = f"'{method}' is not a valid HTTP API method."
         super().__init__(self.message)
+
```

Comment:
You should have two lines before a new class (per [PEP8](https://peps.python.org/pep-0008/#blank-lines)

Reply:
Thank you!